data-plane/pom.xml

```diff
@@ -42,10 +42,10 @@
     <spotless.plugin.version>2.38.0</spotless.plugin.version>
 
     <!-- dependencies version -->
-    <vertx.version>4.5.7</vertx.version>
+    <vertx.version>4.5.11</vertx.version>
     <cloudevents.sdk.version>4.0.0</cloudevents.sdk.version>
     <micrometer.version>1.12.5</micrometer.version>
```

Comment:
I see the import change from
``
import io.micrometer.prometheus.PrometheusConfig;
import io.micrometer.prometheus.PrometheusMeterRegistry;
```
to
```
import io.micrometer.prometheusmetrics.PrometheusConfig;
import io.micrometer.prometheusmetrics.PrometheusMeterRegistry;
```
but the `micrometer.version` is not changing?

Reply:
Ah, yeah. The 1.33.5 is used here, but it comes via quarkus / vertx and their bom. 

I guess we can remove the micrometer-bom import at the root `pom.xml` ? 

For now I have changed it on the root pom, and updated the PR